src/components/navigation/Hamburger/index.js

```diff
@@ -1,23 +1,29 @@
 import { tabsEnum } from 'components/common/Tabs';
 import useTranslation from 'hooks/useTranslation';
-import React, { useCallback } from 'react';
+import React, { useCallback, useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
 import routesPaths from 'routes/routesPaths';
-import { setHomeTab } from 'state/slices/tabSlice';
+import { setHomeTab, setShowContactDialog } from 'state/slices/tabSlice';
 import './styles.scss';
 
 const Hamburger = ({ tab = false }) => {
+  const [checked, setChecked] = useState(false);
   const t = useTranslation();
   const dispatch = useDispatch();
 
   const handleAboutTab = useCallback(() => {
     dispatch(setHomeTab(tabsEnum.about));
   }, [dispatch]);
 
+  const handleContact = useCallback(() => {
+    setChecked(false);
+    dispatch(setShowContactDialog(true));
+  }, [dispatch]);
+
   return (
     <div className="hamburger-container">
-      <input type="checkbox" />
+      <input type="checkbox" checked={checked} onChange={() => setChecked(!checked)} />
       <div className="hamburger-line" />
```

Comment:
Just an advice: I did this hamburger menu with an svg imported from figma. Same as the smile img or the other ones you can import the hamburger icon and just use it instead of coding this divs.

Reply:
Thank you! Will try to apply it in the future